### PR TITLE
Add option to ignore filtered notifications to the web interface

### PR DESCRIPTION
**Pull request by ClearlyClaire**
image
image
image

### CODE REVIEW

**Comment by ClearlyClaire** [comment]
This is now fully implemented:

https://github.com/user-attachments/assets/707d2791-3695-4f23-bc5a-4e29b7fc334d

**Review by Gargron — APPROVED** [review]

**Comment by vmstan** [comment]
This looks great!

**Comment by shleeable** [comment]
Wonderful!

**Comment by sneakers-the-rat** [comment]
NICE! thank you!

File: app/javascript/mastodon/locales/en.json [hunk 1]
```diff
@@ -356,6 +356,17 @@
   "home.pending_critical_update.link": "See updates",
   "home.pending_critical_update.title": "Critical security update available!",
   "home.show_announcements": "Show announcements",
+  "ignore_notifications_modal.disclaimer": "Mastodon cannot inform users that you've ignored their notifications. Ignoring notifications will not stop the messages themselves from being sent.",
+  "ignore_notifications_modal.filter_instead": "Filter instead",
+  "ignore_notifications_modal.filter_to_act_users": "Filtering helps avoid potential confusion",
+  "ignore_notifications_modal.filter_to_avoid_confusion": "Filtering helps avoid potential confusion",
+  "ignore_notifications_modal.filter_to_review_separately": "You can review filtered notifications speparately",
```
Comment (ciencia) [inline app/javascript/mastodon/locales/en.json:363]: typo: speparately

File: app/javascript/mastodon/features/ui/components/ignore_notifications_modal.jsx [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+    <div className='modal-root__modal safety-action-modal'>
+      <div className='safety-action-modal__top'>
+        <div className='safety-action-modal__header'>
+          <h1>{title}</h1>
+        </div>
+
+        <div className='safety-action-modal__bullet-points'>
+          <div>
+            <div className='safety-action-modal__bullet-points__icon'><Icon icon={InventoryIcon} /></div>
+            <div><FormattedMessage id='ignore_notifications_modal.filter_to_review_separately' defaultMessage='You can review filtered notifications speparately' /></div>
```
Comment (ciencia) [inline app/javascript/mastodon/features/ui/components/ignore_notifications_modal.jsx:63]: typo: speparately

**Review by ciencia — COMMENTED** [review]
There's a typo in the messages

File: app/javascript/mastodon/locales/en.json [hunk 3]
```diff
@@ -356,6 +356,17 @@
   "home.pending_critical_update.link": "See updates",
   "home.pending_critical_update.title": "Critical security update available!",
   "home.show_announcements": "Show announcements",
+  "ignore_notifications_modal.disclaimer": "Mastodon cannot inform users that you've ignored their notifications. Ignoring notifications will not stop the messages themselves from being sent.",
+  "ignore_notifications_modal.filter_instead": "Filter instead",
+  "ignore_notifications_modal.filter_to_act_users": "Filtering helps avoid potential confusion",
```
Comment (ciencia) [inline app/javascript/mastodon/locales/en.json:361]: This seems like a duplicate text. The default message reads "You'll still be able to accept, reject, or report users".

**Pull request merged**